src/test/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerCreatePaperCaseTest.java

```diff
@@ -124,4 +125,36 @@ public void shouldNotSetApplicant2OfflineWhenSoleApplicationAndApplicant2HasEmai
 
         assertThat(submitResponse.getData().getApplicant2().getOffline()).isEqualTo(NO);
     }
+
+    @Test
+    public void shouldSetApplicant2OfflineWhenJudicialSeparation() {
+        final CaseData caseData = caseData();
+        caseData.setApplicationType(ApplicationType.SOLE_APPLICATION);
+        caseData.setSupplementaryCaseType(SupplementaryCaseType.JUDICIAL_SEPARATION);
+        caseData.getApplicant2().setEmail("testapp2@test.com");
+
+        final CaseDetails<CaseData, State> details = new CaseDetails<>();
+        details.setData(caseData);
+        details.setId(1L);
+
+        AboutToStartOrSubmitResponse<CaseData, State> submitResponse = caseworkerCreatePaperCase.aboutToSubmit(details, details);
+
+        assertThat(submitResponse.getData().getApplicant2().getOffline()).isEqualTo(YES);
+    }
```

Comment:
Possibly assert app1 is also offline? (picky, can be ignored!)

Reply:
A little, but it's a 2 liner.  Done.